sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,118 @@
+# Mar'24 release notes
+Release notes for commit range [f4e0d3177338](https://github.com/intel/llvm/commit/f4ed132f243ab43816ebe826669d978139964df2).. [d2817d6d317db1](https://github.com/intel/llvm/commit/d2817d6d317db1143bb227168e85c409d5ab7c82)
+
+## New Features
+### SYCL Compiler
+
+- Added more available CPU for `-march` option in OpenCL AOT compiler. [7911773c]
+- Added support for additional AMD GPU targets. [c1ce15944]
+- Added C++11 ABI=0 support. [459e122a]
+- Created an additional version-agnostic copy of the SYCL import library during compiler build. [2d2e418c]
```

Comment:
This seems to be more about packaging than about compiler

Reply:
Perhaps I can add a "Miscellaneous" section for changes like these.